sinks/honeycomb.go

```diff
@@ -26,7 +26,8 @@ func (hs *HoneycombSink) Start() error {
 	for _, v := range hs.DropFields {
 		hs.dropFieldsMap[v] = struct{}{}
 	}
-	libhoney.UserAgentAddition = "honeycomb-opentracing-proxy"
+	versionStr := "1.0.1"
+	libhoney.UserAgentAddition = "honeycomb-opentracing-proxy" + versionStr
```

Comment:
I think this should be in the form of `honeycomb-opentracing-proxy/1.0.1` to match up with the libhoney/beeline version format

![Screen Shot 2019-11-05 at 15 48 30](https://user-images.githubusercontent.com/475364/68256008-bd4b5280-ffe3-11e9-8b6b-6baab37e69e1.png)